src/ast/irbuilderbpf.cpp

```diff
@@ -999,6 +999,16 @@ CallInst *IRBuilderBPF::CreateGetNs(TimestampMode ts, const location &loc)
   return CreateHelperCall(fn, gettime_func_type, {}, "get_ns", &loc);
 }
 
+CallInst *IRBuilderBPF::CreateJiffies64(const location &loc)
+{
+  FunctionType *jiffies64_func_type = FunctionType::get(getInt64Ty(), false);
```

Comment:
Add comment with the helper signature (see other `Create...` functions in `irbuilder.cpp`).